Resources/Locale/en-US/botany/components/plant-holder-component.ftl

```diff
@@ -31,3 +31,4 @@ plant-holder-component-heat-improper-warning = The [color=orange]improper temper
 plant-holder-component-pressure-improper-warning = The [color=lightblue]improper environment pressure alert[/color] is blinking.
 plant-holder-component-gas-missing-warning = The [color=cyan]improper gas environment alert[/color] is blinking.
 plant-holder-component-early-sample-message = The plant hasn't grown enough to take a sample yet.
+plant-holder-component-ligneous-cant-harvest-message = The plant is too tough.
```

Comment:
I think the PR is great QOL, but I would probably make the description hint just a bit more towards the solution.
```suggestion
plant-holder-component-ligneous-cant-harvest-message = The plant is too tough to harvest with your bare hands.
```